tests/x11/gnomecase/change_password.pm

```diff
@@ -143,7 +162,7 @@ sub run {
 
     #delete the added user: test
     # We should kill the active user test in SLE15
-    assert_script_run 'loginctl kill-user test' if (sle_version_at_least('15'));
+    assert_script_run 'loginctl kill-user test' if (is_sle && sle_version_at_least('15'));
```

Comment:
When changing this already I suggest you avoid the deprecated method `sle_version_at_least` and use in this case `is_sle('15+')`